Assets/LeapMotion/Core/Scripts/Streams/IStreamReceiver.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+
+namespace Leap.Unity {
```

Comment:
If there is a Leap.Unity.Streams namespace, I think all these scripts in the Streams folder should probably be in that namespace!

Reply:
agree!